src/ocamlorg_toplevel/bin/dune

```diff
@@ -104,14 +75,27 @@
  js/
  (rule
   (alias toplevel)
-  (targets stdlib-4.13.0.js worker.js worker_base.js base-v0.14.1.js)
+  (targets
+   stdlib-4.13.0.js
+   worker.js
+   worker_base.js
+   base-v0.14.1.js
+   playground.js)
   (deps
+   ../../playground/index.bc.js
    ../toplevel_stdlib.bc.js
    ../toplevel_base.bc.js
    ../worker.js
    ../worker_base.js)
   (action
    (progn
+    (run
+     npx
```

Comment:
I'm surprised you managed to run `npx` there. I always had trouble running it from `dune` 🤷